README.md

```diff
@@ -59,6 +59,7 @@ The **Run MATLAB Tests** action lets you customize your test run using optional
 Input                     | Description
 ------------------------- | ---------------
 `source-folder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. If you specify `source-folder` and then generate a coverage report, MATLAB uses only the source code in the specified folder and its subfolders to generate the report. You can specify multiple folders using a colon-separated or semicolon-separated list.<br/>**Example:** `source`<br/>**Example:** `source/folderA; source/folderB`
+`use-parallel`              | (Optional) Whether to run tests in parallel on a self-hosted runner, specified as `false` or `true`. By default, the value is `false` and tests run in serial. If the test runner configuration is suited for parallelization, a `true` value results in tests being run in parallel. This input requires a Parallel Computing Toolbox™ license, and is supported only on self-hosted runners.
```

Comment:
Like other inputs, should we also say which release is required (R2015a or later)?

Reply:
We can if you don't think it will cause more harm than good. R2015a+ is only true if you produce no artifacts. As soon as you start switching on artifacts, that release changes.

Reply:
Ok, this would be an overkill. Let's keep the text as is.

Reply:
Our editor would probably recommend removing the last comma. Would you like to remove it now?